### PR TITLE
Setup clarifications

### PR DESCRIPTION
**Pull request by jgwerner**
- Docker compose setup with full stack
- Trouble shooting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,27 +64,49 @@ Connect:
...
+[API](https://hub.docker.com/r/3blades/app-backend)
+[OpenResty](https://hub.docker.com/r/3blades/)
+
+Export environment variables listed below with your local settings, for example:
+
+    export C_ROOT=1
+    export DATABASE_URL='postgres://postgres:@db:5432/postgres'
+    export DEBUG=True
+    export DJANGO_SETTINGS_MODULE='appdj.settings.dev'
+    export DOCKER_HOST='DOCKER_HOST=tcp://my_vm_ipv4_address:2375'
```
Comment (jkp85) [inline README.md]: You should remove second DOCKER_HOST from here

File: README.md [hunk 2]
```diff
@@ -96,28 +116,57 @@ Install dev dependencies:
...
+3034d8539114        dev_api                           "/srv/app/docker-e..."   55 minutes ago      Up 53 minutes       80/tcp                                                  vagrant_api_1
+3b2e36ca71f2        postgres:alpine                       "docker-entrypoint..."   55 minutes ago      Up 55 minutes       0.0.0.0:5432->5432/tcp                                  vagrant_db_1
+94cf97bb0bc6        rabbitmq:alpine                       "docker-entrypoint..."   55 minutes ago      Up 55 minutes       4369/tcp, 5671/tcp, 25672/tcp, 0.0.0.0:5672->5672/tcp   vagrant_broker_1
+bd8179fd15b8        3blades/notifications-server:latest   "npm start"              55 minutes ago      Up 55 minutes       0.0.0.0:3000->3000/tcp                                  vagrant_notifications-server_1
+3ef5ab52176b        redis:alpine                          "docker-entrypoint..."   55 minutes ago      Up 55 minutes       0.0.0.0:6379->6379/tcp                                  vagrant_cache_1
+c06b9d4c73ae        gliderlabs/logspout                   "/bin/logspout"          55 minutes ago      Up 55 minutes       80/tcp                                                  vagrant_logspout_1
+```
+
```
Comment (AbhishikthaReddy) [inline README.md]: python manage.py migrate, python manage.py runserver 0.0.0.0:8000 instructions are missing in Dev Setup with Docker. Everything looks okay apart from that.

Comment (jgwerner) [inline README.md]: python manage.py migrate is run automatically with bash script from docker image CMD, so its not necessary for docker-compose up option. The runserver command is implicit with CMD as well.

Comment (AbhishikthaReddy) [inline README.md]: true, my mistake

**Pull request merged**